script/Common/00_02.txt

```diff
@@ -276,17 +276,17 @@ A stranger emerges from the other side.
 [sha:cd6a1bcbda2e62da47b168b0e9e4158e99eb080c]{
 -- Page 12, Offset 61.
 -- 凶器を手にしてやってくる。
-Coming this way with a deadly weapon in hand.
+He approaches me, a deadly weapon in his hand.
 }
 [sha:7b655f0c2b514e52bf47c9aeb15beac970645160]{
 -- Page 12, Offset 62.
 -- みんなをそうしたように　ボクもバラバラにしたいみたい。
-He looks like he wants to tear me apart, just like everyone else.
+He looks like he wants to tear me apart, just as he did everyone else.
```

Comment:
i'm torn on this one because i think the old version sounds cooler but i also see what you're getting at. it does sound a _little_ like everyone wants to tear shiki apart but i still think revert.

Reply:
we discussed this quite a long time ago, but the problem with doing it like _that_ is that it's not like... childish, it just reads weird. It _needs_ to be editorialized with some verb or other. Maybe something like this:
> He looks like he wants to tear me to bits, just like everyone else.

Reply:
The problem is that the alternative destroys the intent of the author.
Bits and pieces read pretty awkward, I'd be foolish to say I love it: but at least it respects that part of the scene.
That's why I mean its important. I don't think you can just go 'well, we needed something that read alright', it's not that light of a creative decision.

Other suggestions:
'Take me apart'
'Pick me apart'
'Break me into pieces'

Reply:
"Take me apart" is good.

Reply:
Not sure I agree that `rip/tear me apart` 'destroys the intent of the author', I feel like I wouldn't really be shocked to hear kids this age using that phrase while playing with their action men or whatever. Rip probably more childish that tear though, I guess `rip me apart` would be my preference. `break me into pieces` maybe second choice.

Reply:
It's less about the childishness, but as I said above, he keeps going "よく　わからない" and so on, so he's quite literally unable to grasp that people are dying. It'd be strange in the context of the scene that he'd then use a word that is so obviously "violence". Or well, that's how I'd feel about it.

Reply:
I guess I read the わからない more as _why_ rather than _what_. Like he understands things like "mom's dead" and 凶器, so while I agree we probably shouldn't be _too_ explicit I think that the violence of ripping isn't over the edge?

Reply:
He sort of understands his mom is dead, in a very abstract way, seemingly. I think his 'detachment' to the violence is at least quite striking when you read it. 

But sure, 'tear apart' isn't the most evocatively violent thing you could say, it just felt more explicit than バラバラ is.